src/main/kotlin/uk/gov/justice/hmpps/offenderevents/service/OffenderEventStore.kt

```diff
@@ -16,7 +17,8 @@ import uk.gov.justice.hmpps.offenderevents.service.OffenderEventStore.Topics.UNK
 @Service
 class OffenderEventStore(
   @Value("\${model.cacheSize}") private val cacheSize: Int,
-  val store: MutableList<DisplayMessage> = mutableListOf()
+  val eventRepository: EventRepository,
+  val store: ArrayDeque<DisplayMessage> = ArrayDeque(),
```

Comment:
Change the local event store to a Deque to make it obvious what's happening. I considered using Java's ConcurrentLinkedDeque which should have handled the multi-threaded queue listener, but opted for simplicity instead.